mxcube3/ho_mediators/beamline_setup.py

```diff
@@ -383,10 +413,8 @@ def limits(self):
         :returns: The limits.
         """
         try:
-            energy_limits = self._ho.getLimits()
+            energy_limits = self._ho.getEnergyLimits()
```

Comment:
I would say return the wavelength limits in A instead of eV. How is it done now?